### ISSUE TITLE
Feature: Easy all In One install Script

### ISSUE THREAD

**Issue by johnnyq** [cupcakearmy/cryptgeon#23]
A simple one liner copy and paste code that downloads a script and executes everything it needs to to get Cryptgeon up and running with a valid Let Encrypt cert and good default settings. Compatible with Debian 11 and Ubuntu 20.04

Basically a set it and forget it

This should help people get this up and running much quicker

I love this project You did great work and its very clean =]

**Comment by cupcakearmy** [comment]
Thanks for the feedback!

Like the idea, however this is outside of the scope for this project, the docker-compose file is already that if you couple that with something like traefik.
If you have a look at the [traefik setup in the readme](https://github.com/cupcakearmy/cryptgeon#traefik-2) that is all you have to do for getting certificates, etc. 

Adding the whole certificate and manual setup stuff in the project will add complexity which I'm not willing to deal with tbh because every system will be different.

Basically the install steps are:
1. install docker
2. get traefik running on docker.
3. copy paste docker-compose file and run `docker compose up -d`.

I could see putting a guide to that in the `example` folders. but not a script.

**Comment by cupcakearmy** [comment]
Added a guide https://github.com/cupcakearmy/cryptgeon/tree/main/examples/scratch
